extensions/ql-vscode/src/cli.ts

```diff
@@ -1216,6 +1217,23 @@ export class CodeQLCliServer implements Disposable {
     );
   }
 
+  /**
+   * Adds a list of QL library packs with optional version ranges as dependencies of
```

Comment:
Minor: the jsdoc isn't quite right.

```suggestion
   * Adds a core language QL library pack for the given query language as a dependency of
```